lib/pages/project/project_page.dart

```diff
@@ -908,9 +908,17 @@ class _ProjectPageState extends State<ProjectPage> {
         for (PathPlannerPath path in _paths) {
           pathNames.add(path.name);
         }
-        String pathName = 'Copy of ${_paths[i].name}';
+        String pathName = _paths[i].name;
+        RegExp exp = RegExp(r'\(\d+\)');
+        String source = pathName.substring(pathName.length - 3);
         while (pathNames.contains(pathName)) {
-          pathName = 'Copy of $pathName';
+          if (exp.hasMatch(source)) {
```

Comment:
I'm assuming tests are failing because you're matching the regex against the substring with the `(x)` removed. I think this should be changed to match against the entire path name and then only do the substring if it has a match. Same thing for the autos.

Reply:
When fixing this, could you also add on to the duplication tests to duplicate them a third time. i.e. check for `Example Path (3)` just to be extra sure the increment works.